[2023-03-16 12:06:21,035] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: attack_dag2.trigger_defence scheduled__2023-03-15T00:00:00+00:00 [queued]>
[2023-03-16 12:06:21,045] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: attack_dag2.trigger_defence scheduled__2023-03-15T00:00:00+00:00 [queued]>
[2023-03-16 12:06:21,046] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-03-16 12:06:21,046] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-03-16 12:06:21,047] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-03-16 12:06:21,060] {taskinstance.py:1377} INFO - Executing <Task(TriggerDagRunOperator): trigger_defence> on 2023-03-15 00:00:00+00:00
[2023-03-16 12:06:21,064] {standard_task_runner.py:52} INFO - Started process 87 to run task
[2023-03-16 12:06:21,067] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'attack_dag2', 'trigger_defence', 'scheduled__2023-03-15T00:00:00+00:00', '--job-id', '1245', '--raw', '--subdir', 'DAGS_FOLDER/attack.py', '--cfg-path', '/tmp/tmp9eskpjje', '--error-file', '/tmp/tmp917_puzm']
[2023-03-16 12:06:21,067] {standard_task_runner.py:80} INFO - Job 1245: Subtask trigger_defence
[2023-03-16 12:06:21,121] {task_command.py:369} INFO - Running <TaskInstance: attack_dag2.trigger_defence scheduled__2023-03-15T00:00:00+00:00 [running]> on host 596adcca7378
[2023-03-16 12:06:21,193] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=attack_dag2
AIRFLOW_CTX_TASK_ID=trigger_defence
AIRFLOW_CTX_EXECUTION_DATE=2023-03-15T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-03-15T00:00:00+00:00
[2023-03-16 12:06:22,310] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=attack_dag2, task_id=trigger_defence, execution_date=20230315T000000, start_date=20230316T120621, end_date=20230316T120622
[2023-03-16 12:06:22,364] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-03-16 12:06:22,398] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
